twisted/conch/test/test_ckeygen.py

```diff
@@ -81,12 +106,12 @@ def test_saveKey(self):
         base.makedirs()
         filename = base.child('id_rsa').path
         key = Key.fromString(privateRSA_openssh)
-        _saveKey(key, {'filename': filename, 'pass': 'passphrase'})
+        _saveKey(key, {'filename': filename, 'pass': 'passphrase', 'format': 'md5-hex'})
```

Comment:
I think not naming the dictionary is alright because that has been followed all through this module.
Example : https://github.com/twisted/twisted/blob/trunk/twisted/conch/test/test_ckeygen.py#L84,
https://github.com/twisted/twisted/blob/trunk/twisted/conch/test/test_ckeygen.py#L221,
https://github.com/twisted/twisted/blob/trunk/twisted/conch/test/test_ckeygen.py#L158